qcodes/config/__init__.py

```diff
@@ -1,2 +1,27 @@
+import json
+
 from .config import Config
 from .config import logger
+from .config import DotDict
+
+cfg = Config()
+loc = cfg['GUID_components']['location']
+station = cfg['GUID_components']['work_station']
+
+if station == 0 or loc == 0:
+    print('Welcome to QCoDeS!\nTo use QCoDeS, please provide the system with '
+          'enough information to generate unique dataset identifiers.')
```

Comment:
how non-ms-collab users know what defaults/numbers they can/should use?

Reply:
Could/should we move this into the dataset

Reply:
as discussed, please move to the "dataset"